scss/compounds/_post-card.scss

```diff
@@ -0,0 +1,9 @@
+.post-card {
+  figure { 
+    & ~ h3 {
```

Comment:
we're going to have only 1 H3 for each post-card so please correct the code accordingly